tasks/setup-repository.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: Include setup repository tasks for distribution variety {{ _docker_os_dist_file_varity }}
+  when: (_docker_os_dist == "Amazon" and _docker_os_dist_major_version | int < 2023) or _docker_os_dist != "Amazon"
```

Comment:
Just question here, have they changed in Amazon image 2023 so that previously Docker related repos for newest Docker release were available?

Reply:
I didn't check what is going on with repos in there, but they provide an easy way to install docker-ce. You can install docker there without adding new repos `dnf install -y docker`.

Reply:
I read some official documentation and seems no mention of adding repos for any of the Amazon images but I will fix this after I merge your changes.

For reference which mention about the Amazon image: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-docker.html